templates/vsc/js/basic-custom-engine-agent/.vscode/tasks.json

```diff
@@ -299,6 +300,15 @@
             "args": {
                 "url": "teams.microsoft.com/l/app/${{TEAMS_APP_ID}}?installAppPackage=true"
             }
+        },
+        {
+            "label": "Sideload App to channel",
```

Comment:
Is this used for dev sandbox? Should we add it in this fix

Reply:
I copied this from TS template to align with it. It was just fixed by another PR.